api/v1alpha1/flowcollector_types.go

```diff
@@ -143,6 +143,13 @@ type FlowCollectorIPFIX struct {
 	OVNKubernetes OVNKubernetesConfig `json:"ovnKubernetes,omitempty" mapstructure:"-"`
 }
 
+type FeatureConfigType string
```

Comment:
sorry there's a misunderstanding, I was thinking about an enum like this here:

```
// +kubebuilder:validation:Enum:="PKT_DROP";"DNS_TRACKING";"FLOW_RTT"
type AgentFeature string

const (
	PktDrop     AgentFeature = "PKT_DROP"
	DNSTracking AgentFeature = "DNS_TRACKING"
	FlowRTT     AgentFeature = "FLOW_RTT"
)
```

And below, in ebpf agent spec:
```
	Features []AgentFeature `json:"features,omitempty"`
```

Reply:
@jotak Ah I see..I misunderstood that you wanted strings instead of bool as was present for other examples. 
I think I can take a stab at this change if not ask you or @jpinsonneau for help.